handlers/message.js

```diff
@@ -8,6 +8,7 @@ const {
     }
   }
 } = require("../utilities/bolt.js");
+const { getMarkdownSection, getActionButton } = require("../utilities/helperFunctions");
```

Comment:
Please rename these to `buildMarkdownSection` or `makeActionButton` or something similar to better describe its true purpose. We're not "get"ing anything, we're using them as a creator.

Reply:
Or maybe `genMarkdownSection` for "generate." Idk, I'll leave the nitty gritty up to you :smiley:

Reply:
Thought process behind using "get" was because a value is being obtained, but can do.